### ISSUE TITLE
is jjwt include jjwt-impl & jjwt-api and the others

### ISSUE THREAD

**Issue by linyongfu2013** [jwtk/jjwt#526]
i see a io.jsonwebtoken:jjwt jar in maven central, is this jar include the others jjwt-xxx jar ?

in other words, can only use io.jsonwebtoken:jjwt jar file in my project ?

**Comment by lhazlewood** [comment]
Please use JJWT 0.10.7 which has the appropriate api and impl jars.  `io.jsonwebtoken:jjwt` is for old (less than 0.10.0) releases and not recommended.

In the future, please do not use GitHub Issues to ask usage questions:

https://github.com/jwtk/jjwt#help-questions

**Comment by l3ender** [comment]
It might be helpful if this was in the readme--searching Maven Repository for "jjwt" brings up `io.jsonwebtoken:jjwt` as the first result. After checking the readme, I also wasn't sure (even though my project was building/running as expected).

**Comment by lhazlewood** [comment]
How is https://github.com/jwtk/jjwt#install not already clear enough?  (Not being argumentative - genuinely curious).

**Comment by l3ender** [comment]
A lot of project readme/installation doc don't reference the latest version of their artifact(s), so as a result of habit, I always go to Maven Central to see if there are any newer versions. When I did that for this project and saw the singular `jjwt` version, I went back to the readme to search for it and didn't find anything.

Dropping a note under https://github.com/jwtk/jjwt#understanding-jjwt-dependencies would explain that it's an old dependency and shouldn't be used.
